lib/src/functions/map.dart

```diff
@@ -43,7 +43,7 @@ final _get = _function("get", r"$map, $key, $keys...", (arguments) {
 final _set = BuiltInCallable.overloadedFunction("set", {
   r"$map, $key, $value": (arguments) {
     var map = arguments[0].assertMap("map");
-    return _setImpl(map, [arguments[1]], arguments[2]);
+    return _modify(map, [arguments[1]], (oldValue) => arguments[2]);
```

Comment:
```suggestion
    return _modify(map, [arguments[1]], (_) => arguments[2]);
```

This helps make it clear at a glance that the argument isn't used.

Reply:
I was hoping there was a better way to do this :)